NewsApp/NewsApp/View/ThematiqueCell.swift

```diff
@@ -13,5 +13,9 @@ class ThematiqueCell: UITableViewCell {
 
     func configure(category: Category) {
         self.thematique.text = category.title
+        guard let systemImage = category.image else {
+            return
+        }
+        imageThematique.image = UIImage(systemName: systemImage)
```

Comment:
préféré un if let plutôt que guard let ici je pense

Reply:
fix in "correction PR"